clients/app/src/views/components/base/DateTimePicker/PickerWrapper.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { Box, Pressable, Text } from 'native-base';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import { OnChangeInterface, PickerWrapperProps } from './types';
+
+const PickerWrapper: React.FC<PickerWrapperProps> = ({
+  autoclose = true,
+  text,
+  value,
+  onChange,
+  ...rest
+}) => {
+  const [opened, setOpened] = React.useState<boolean>(true);
+
+  const onValueChange: OnChangeInterface = React.useCallback(
+    (event, date) => {
+      if (typeof onChange === 'function') {
+        onChange(event, date);
+      }
+
+      if (autoclose) {
+        setOpened(false);
+        setTimeout(() => {
```

Comment:
It's not clear what is the purpose of setTimeout?
Could you please explain?

Reply:
The package we are using doesn't contain prop to make it auto-closable,

previously datepicker component were transparent and upper text component, and on pressing it datepicker was shown,

now I've added a condition to render datepicker component, on date change the flag was set to false and the component will be removed from React hierarchy, after 10 milliseconds I changed it back to true, to add a transparent component back to React hierarchy,

currently, I've no idea how to make an auto-close simulation.